src/Hydration.js

```diff
@@ -1,23 +1,32 @@
+import {averager,  findDataByDate,  finder}  from './util';
+
 class Hydration {
   constructor(hydrationData) {
     this.hydrationData = hydrationData;
   }
+
   calculateAverageOunces(id) {
-    let perDayUserHydration = this.hydrationData.filter((data) => id === data.userID);
-    return perDayUserHydration.reduce((sumSoFar, data) => {
-      return sumSoFar += data.numOunces;
-    }, 0) / perDayUserHydration.length;
+    let perDayUserHydration = this.hydrationData.filter(data => id === data.userID);
+    return averager(perDayUserHydration, `numOunces`);
```

Comment:
Im enjoy that you were able to combine multiple functions by enlisting the util.js file. 
One thing about Averager, is that its not a word, and if it was, it would be a noun. All methods and functions should be verbs. Perhaps, findAverage or calcAverage might be more appropriate.